modules/http/scanner.go

```diff
@@ -280,9 +280,9 @@ func (scanner *Scanner) newHTTPScan(t *zgrab2.ScanTarget) *scan {
 	ret.client.Transport = ret.transport
 	ret.client.Jar = nil // Don't send or receive cookies (otherwise use CookieJar)
 	ret.client.Timeout = scanner.config.Timeout
-	host := t.Domain
-	if host == "" {
-		host = t.IP.String()
+	host := t.IP.String()
```

Comment:
Isn't this going to crash now if `t.IP` is nil?

Reply:
No, in that case IP.String() actually returns the string `"<nil>"`